core/src/main/java/org/sonarsource/sonarlint/core/container/connected/update/ModuleConfigUpdateExecutor.java

```diff
@@ -120,6 +117,11 @@ private void updateStatus(Path temp) {
     ProtobufUtil.writeToFile(updateStatus, temp.resolve(StorageManager.UPDATE_STATUS_PB));
   }
 
+  private void fetchModuleHierarchy(String moduleKey, ModuleConfiguration.Builder builder) {
+    Map<String, String> moduleHierarchy = moduleHierarchyDownloader.fetchModuleHierarchy(moduleKey);
+    moduleHierarchy.forEach((key, path) -> builder.getMutableModulesKeysByPath().put(path, key));
```

Comment:
Wouldn't it be better to extract `builder.getMutableModulesKeysByPath()` to a variable before the `forEach` ?